oak_functions_containers_app/src/lib.rs

```diff
@@ -212,12 +213,49 @@ impl<G: AsyncRecipientContextGenerator + Send + Sync + 'static> OakFunctions
     }
 }
 
+fn create_trace<Body>(request: &http::Request<Body>) -> Span {
+    let uri = request.uri();
+    let mut parts = uri.path().rsplitn(3, "/");
```

Comment:
Could you add a comment about what this is for (perhaps give an example of a gRPC method URL).